### DISCUSSION TITLE
Solid JS Support

### DISCUSSION THREAD

**Discussion by fbritoferreira** [trpc/trpc#1364] [General]
Is solidjs support something that is coming?

**Comment by sachinraja** [comment]
@TheoBr managed to get trpc working with solidjs at [TheoBr/solid-trpc](https://github.com/TheoBr/solid-trpc). There doesn't seem to be much to the implementation, looks like it's just [this file](https://github.com/TheoBr/solid-trpc/blob/main/src/lib/trpc.ts).

**Reply by t3dotgg** [reply]
Yep, wasn't too hard at all :) The addition of a mutation primitive in the near future within Solid will make this even easier

**Comment by OrJDev** [comment]
it is not perfect but it should be working fine, try [create-jd-app](https://github.com/OrJDev/create-jd-app)

Edit:
Just created SolidJS tRPC package based on the new `@tanstack/solid-query`: [solid-trpc](https://github.com/OrJDev/solid-trpc).
